packages/app/src/utils/oas/index.ts

```diff
@@ -786,6 +786,10 @@ export const mergeTableSortRequestValue = (
         break;
     }
   }
+  // HOTFIX: 空の配列をふくまないようにする
+  if (!requestValue.parameters?.sort.length) {
+    delete requestValue.parameters?.sort;
+  }
```

Comment:
When sort is none, query is as follows:
`?sort=`
In this case, the API may not support it, so it must be removed.